clang/test/AST/ast-attr-add-ir-annotations-pack.cpp

```diff
@@ -0,0 +1,848 @@
+// RUN: not %clang_cc1 -fsycl-is-device -std=gnu++11 -ast-dump %s | FileCheck %s
```

Comment:
Is gnu std important for this test?

Reply:
Doesn't seem like it. It has been removed.